app/models/comment.rb

```diff
@@ -248,8 +248,19 @@ def delete_for_user(user, reason = nil)
     self.user.refresh_counts!
   end
 
-  def deliver_mention_notifications
+  def deliver_notifications
+    notified = deliver_reply_notifications
+    deliver_mention_notifications(notified)
+  end
+
+  def deliver_mention_notifications(notified = nil)
```

Comment:
You can set `notified=[]`, Ruby doesn't have the Python gotcha about list defaults in args.